clang/lib/Interpreter/Wasm.cpp

```diff
@@ -72,13 +72,13 @@ llvm::Error WasmIncrementalExecutor::addModule(PartialTranslationUnit &PTU) {
   OutputFile.close();
 
   std::vector<const char *> LinkerArgs = {"wasm-ld",
-                                          "-pie",
+                                          "-shared",
                                           "--import-memory",
                                           "--no-entry",
                                           "--export-all",
                                           "--experimental-pic",
-                                          "--no-export-dynamic",
```

Comment:
This change has been made cause `-shared` by default invokes `--export-dynamic`.
Not a breaking change that harms us (but surely we would want `shared` instead of `pie`)